services/blockchain-indexer/shared/dataService/business/blockchainApps.js

```diff
@@ -0,0 +1,82 @@
+/*
+* LiskHQ/lisk-service
+* Copyright © 2022 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*
+*/
+const { MySQL: { getTableInstance } } = require('lisk-service-framework');
```

Comment:
Please refactor the file into separate files, one for the apps information and the other for the stats. Please add them to the `interoperability` (module specific) directory. Please ensure that the module-based separation is followed all across the codebase.